src/joint_state_listener.cpp

```diff
@@ -152,7 +152,9 @@ int main(int argc, char** argv)
 
   // gets the location of the robot description on the parameter server
   urdf::Model model;
-  model.initParam("robot_description");
```

Comment:
If initialization of model failed, `initParam()` returns false, which was ignored here.